### ISSUE TITLE
Purge requests for `__home__` instead of `/`

### ISSUE THREAD

**Issue by kbergha** [putyourlightson/craft-blitz#673]
### Support Request

Hi.

I'm trying to use Blitz for purging with Cloudfront.
When saving the home page entry I'm getting purge requests for `/__home__/` instead of the expected `/`

PHP: 8.3.8
Craft: 5.0.3
Blitz: 5.3.1
Blitz CloudFront Purger: 5.0.0

### Blitz setup

As of right now, I'm trying to use Blitz as a purger only due to the issue I have with the local generator (#672)

![2024-06-17 16 07 18 15139e0006e3](https://github.com/putyourlightson/craft-blitz/assets/4321846/3894d342-b8c3-4bbc-bc6d-0129e844b7ed)

### Purger setup

![2024-06-17 15 50 32 craft cic oslo kommune localhost 347e61c8a351](https://github.com/putyourlightson/craft-blitz/assets/4321846/37cdea83-cc00-4d24-b4de-bc064cd1b6af)

### Invalidation request from AWS console

![2024-06-17 14 51 35 us-east-1 console aws amazon com fac23373542b](https://github.com/putyourlightson/craft-blitz/assets/4321846/c1da352b-2069-45ba-a163-ec8dfdd26a0e)

### Section set up

You can see that this section is selected as the home page:

![2024-06-17 15 46 34 6aba47b399dd](https://github.com/putyourlightson/craft-blitz/assets/4321846/feb9cc02-53ea-4aba-a6d6-579dae30de99)

### A test query from the database

I'm guessing this is where `/__home__/` comes from.

```
mysql> select id, title, uri from elements_sites where slug = 'forsiden' order by id desc limit 1\G;
*************************** 1. row ***************************
   id: 64
title: Forsiden
  uri: __home__
1 row in set (0.00 sec)
```

### Summary

Why do I get  `/__home__/` instead of the expected `/` in the invalidation / purge request in AWS ?
This is probably related to the change done in https://github.com/putyourlightson/craft-blitz/issues/538 ?

I didn't open this issue in the CloudFront purger repo, because I don't think it's a CloudFront specific issue.
It's probably an issue with `SiteUriHelper` and/or `SiteUriModel` in the base plugin.

### Plugin Version

5.3.1

**Comment by bencroker** [comment]
Thanks for reporting this. The addition in https://github.com/putyourlightson/craft-blitz/commit/385ab48749efdf5ef0dd63cd2ae33f69834b6e11 should fix this. Can you please test and let me know if it resolves the issue for you?

**Comment by kbergha** [comment]
Thanks. 

Using `dev-develop`, it works as expected now.
I also had to update Craft to 5.2.1 at the same time.

![2024-06-18 09 09 53 us-east-1 console aws amazon com 1c6613a7e1d5](https://github.com/putyourlightson/craft-blitz/assets/4321846/08adb76b-c5ee-416f-a939-fddbcee6f48d)

**Comment by bencroker** [comment]
Great! Released in [4.18.2](https://github.com/putyourlightson/craft-blitz/releases/tag/4.18.2).
